editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1203,7 +1205,10 @@ bool CanvasItemEditor::_gui_input_zoom_or_pan(const Ref<InputEvent> &p_event, bo
 				view_offset.y += int(EditorSettings::get_singleton()->get("editors/2d/pan_speed")) / zoom * b->get_factor();
 				update_viewport();
 			} else {
-				_zoom_on_position(zoom * (1 - (0.05 * b->get_factor())), b->get_position());
+				float new_zoom = _get_next_zoom_value(-1);
+				if (b->get_factor() != 1.f)
```

Comment:
We now enforce the use of braces around single-statement `if` blocks, so please update the various `if` checks while rebasing.